core/src/test/scala/unit/kafka/server/ReplicationQuotasTest.scala

```diff
@@ -299,7 +299,7 @@ class ReplicationQuotasTest extends QuorumTestHarness {
     features.add(new BrokerRegistrationRequestData.Feature()
       .setName(MetadataVersion.FEATURE_NAME)
       .setMinSupportedVersion(MetadataVersion.IBP_3_0_IV1.featureLevel())
-      .setMaxSupportedVersion(MetadataVersion.IBP_3_8_IV0.featureLevel()))
+      .setMaxSupportedVersion(MetadataVersion.latestTesting().featureLevel()))
```

Comment:
This makes the test pass as it disregards the latest version as per https://github.com/apache/kafka/pull/16347